src/events/scraper/run-scraper/scraper-helpers/build-get-iso2-from-name.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert')
+const slugify = require('slugify')
+const iso2 = require('country-levels/iso2.json')
+
+const UNASSIGNED = '(unassigned)'
+
+const slugifyOptions = { lower: true }
+/**
+ * @param {object} options - Options for the resultant function.
+ * @param {string} options.country - The iso1 country code to match within (iso1:AU)
+ * @returns {function} - A function that returns the iso2 value when given a name.
+ */
+const buildGetIso2FromName = ({ country }) => {
```

Comment:
Do you feel this method needs a test, or is it good enough as-is?  Not suggesting a test is needed, just curious.

Reply:
Running it against the scrapers is showing that it works, I'll try to write some tests in a subsequent PR though (I like tests too :) )